afinn/data/AFINN-no-165.txt

```diff
@@ -0,0 +1,3214 @@
+truet	-2
+god	3
+fortryllende	3
+rot	-2
+rote til	-2
+metodisk	2
+tankeløs	-2
+mirakel	4
+munterhet	3
+dårlig oppførsel	-2
+ugagn	-1
+feilklassifiserer	-2
+stort	-3
+elendighet	-2
+feiltenning	-2
+feilinformert	-2
+mistolket	-2
+villede	-3
+forlegge	-2
+feilplassert	-2
+forlegger	-2
+feilprising	-3
+feilrapportering	-2
+feilrepresentere	-2
+feilaktig fremstilling	-2
+fremstillinger	-2
+misrepresented	-2
+uriktig	-2
+uriktig fremstilling	-2
+gå glipp av	-2
+savnet	-2
+mista	-2
+misforstå	-2
+misforståelse	-2
+misforstår	-2
+misforstått	-2
+misbruk	-2
+håne	-2
+gjøre narr av	-2
+modern	2
+modernisert	2
+moderniserer	2
+modernisering	2
+monopolisere	-2
+monopolisert	-2
+monopoliser	-2
+humørsyk	-1
+sturer	-1
+jævel	-5
+motivere	1
+motivert	2
+motiverende	2
+motivasjon	1
+sørge over	-2
+sørget	-2
+sørgmodig	-2
+gjørmete	-2
+mord	-2
+morder	-2
+morderisk	-3
+skummel	-2
+myte	-1
+naiv	-2
+narsissisme	-2
+ekkel	-3
+naturlig	1
+trengende	-2
+negativ	-2
+negativitet	-2
+forsømmelse	-2
+forsømt	-2
+neglisjerer	-2
+nerver	-1
+nervøs	-2
+nervøst	-2
+nei	-1
+ikke moro	-3
+edel	2
+edleste	2
+støyende	-1
+ikke-godkjent	-2
+nybegynner	-2
+ikke bra	-2
+jobber ikke	-3
+bemerkelsesverdig	2
+merkbar	2
+notorisk	-2
+roman	2
+nummen	-1
+pleie	2
+nøtter	-3
+utslette	-2
+utslettet	-2
+ytterst ubehagelig	-3
+obskøn	-2
+obskønitet	-2
+besatt	2
+foreldet	-2
+hindringer	-2
+sta	-2
+hinder	-2
+hindrer	-1
+merkelig	-2
+lovbrudd	-2
+fornærme	-2
+lovbryteren	-2
+uakseptable	-2
+støtende	-2
+offensivt	-2
+oks	2
+illevarslende	3
+en gang i livet	3
+muligheter	2
+mulighet	2
+undertrykt	-2
+undertrykkelse	-2
+undertrykkelser	-2
+undertrykkende	-2
+optimisme	2
+optimistisk	2
+utstøtt	-2
+au	-2
+utbrudd	-2
+ramaskrik	-2
+utmanøvrert	-2
+mindretall	-2
+opprør	-3
+oppsøk	2
+fremragende	5
+overbelastning	-1
+oversett	-1
+overbeskyttende	-2
+erobret	-2
+reagere	-2
+overreagerte	-2
+reagerer	-2
+overreaksjon	-2
+sell	-2
+sells	-2
+forglemmelse	-1
+forenkling	-2
+unyansert	-2
+overforenkle	-2
+solgt	-2
+overdriv	-2
+overvekt	-1
+overspent	-3
+smertelig	-2
+smertefull	-2
+panikk	-2
+fikk panikk	-3
+paradis	3
+paradoks	-1
+benådet	2
+tilgivelse	2
+benådninger	2
+lidenskap	1
+lidenskapelig	2
+passiv	-1
+passivt	-1
+patetisk	-2
+betale	-1
+fred	2
+fredelig	2
+straffe	-2
+straffet	-2
+straffer	-2
+penalising	-2
+straff	-2
+tankefull	-1
+perfekt	3
+perfeksjonert	2
+fullkommenhet	3
+perfeksjonerer	2
+peril	-2
+mened	-3
+begått	-2
+gjerningspersonen	-2
+gjerningsmenn	-2
+rådvill	-2
+forfølge	-2
+forfulgt	-2
+forfølger	-2
+opprørt	-2
+pervers	-3
+pessimisme	-2
+pessimistisk	-2
+forsteinet	-2
+filantropi	2
+fobiske	-2
+pittoreske	2
+hoper seg opp	-1
+plyndring	-2
+pirret	-2
+tiss	-4
+ynkelig	-2
+synd	-2
+medlidenhet	-2
+pest	-3
+leken	2
+vær så snill	1
+fornøyd	2
+behage	3
+gripende	2
+meningsløst	-2
+gift	-2
+forgiftet	-2
+forgifter	-2
+polert	2
+forurenseren	-2
+forurensere	-2
+forurensing	-2
+dårligere	-2
+fattigste	-2
+populær	3
+popularitet	3
+positiv	2
+positivt	2
+eiendomspronomen	-2
```

Comment:
Does this make sense as a word with sentiment?

Reply:
I'll remove 'eiendomspronomen'